scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -772,6 +774,36 @@ public final synchronized void destroy() {
 		connectionStore.clear();
 	}
 
+	public void startForEach(LeastRecentlyUsedCache.Predicate<Connection> handler) {
```

Comment:
My intention was to express, that it only starts something. On return, the job is not finished. The implementation hands over the iteration to each serial executor, so it may take some time to finish.

Reply:
The javadoc and returning a Future is a good indication too, but there is no problem, if you keep the `start` prefix :-)